notification/package.json

```diff
@@ -2,10 +2,10 @@
     "name": "open-data-service-notification",
     "version": "1.0.0",
     "description": "TBD",
-    "main": "dist/index.js",
+    "main": "dist/src/index.js",
     "scripts": {
         "start": "npm run transpile && npm run start:transpiled",
-        "start:transpiled": "node dist/index.js",
+        "start:transpiled": "node dist/src/index.js",
```

Comment:
Only `dist/index.js` required if tests are under `src`

Reply:
This was mandatory because with the new "tests" folder the compiler output will create two folders [ "tests", "src" ]